client.go

```diff
@@ -294,6 +295,27 @@ func (c *Client) SetAuthToken(token string) *Client {
 	return c
 }
 
+// SetAuthScheme method sets the auth token scheme type in the HTTP request. For Example:
+//      Authorization: <auth-scheme-value-set-here> <auth-token-value>
```

Comment:
s/-value-set-here//
s/-value//

Reply:
Done in my branch will push it to rigrassm:master